src/test/java/org/graylog/integrations/notifications/types/SlackEventNotificationTest.java

```diff
@@ -145,26 +141,78 @@ public void getCustomMessageModel() {
     }
 
 
-    @Test (expected = PermanentEventNotificationException.class)
+    @Test(expected = PermanentEventNotificationException.class)
     public void execute_with_invalid_webhook_url() throws TemporaryEventNotificationException, PermanentEventNotificationException {
-       //has an invalid webhook url
-       slackEventNotification.execute(eventNotificationContext);
+        //has an invalid webhook url
+        slackEventNotification.execute(eventNotificationContext);
     }
 
 
     @Test
     public void buildCustomMessage() throws PermanentEventNotificationException {
-       String s =  slackEventNotification.buildCustomMessage(eventNotificationContext,slackEventNotificationConfig,"${thisDoesnotExist}");
-       assertThat(s).isEmpty();
-       String expectedCustomMessage =  slackEventNotification.buildCustomMessage(eventNotificationContext,slackEventNotificationConfig,"test");
-       assertThat(expectedCustomMessage).isNotEmpty();
+        String s = slackEventNotification.buildCustomMessage(eventNotificationContext, slackEventNotificationConfig, "${thisDoesnotExist}");
+        assertThat(s).isEmpty();
+        String expectedCustomMessage = slackEventNotification.buildCustomMessage(eventNotificationContext, slackEventNotificationConfig, "test");
+        assertThat(expectedCustomMessage).isNotEmpty();
 
     }
 
     @Test(expected = PermanentEventNotificationException.class)
     public void buildCustomMessage_with_invalidTemplate() throws EventNotificationException {
         slackEventNotificationConfig = buildInvalidTemplate();
-        slackEventNotification.buildCustomMessage(eventNotificationContext,slackEventNotificationConfig,"Title:       ${does't exist}");
+        slackEventNotification.buildCustomMessage(eventNotificationContext, slackEventNotificationConfig, "Title:       ${does't exist}");
+    }
+
+
+    @Test
+    public void test_customMessage_With_Message_Backlog_Override() throws PermanentEventNotificationException {
+
+        SlackEventNotificationConfig slackConfig = SlackEventNotificationConfig.builder()
+                .backlogSize(5)
+                .build();
+        String message = slackEventNotification.buildCustomMessage(eventNotificationContext, slackConfig, "Ich spreche Deutsch");
+        assertThat(message).isEqualTo("Ich spreche Deutsch");
+    }
```

Comment:
What is this actually testing?  How is the behavior you're validating related to the backlog size override?